src/components/beerdetail/BeerDetail.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+
+
+export default function BeerDetail(props) {
+
+    const id = props.match.params.idBeer
+    const beersArray = props.beers
+    const beerDetails = beersArray.find(beer => beer._id === id)
```

Comment:
Once you have the id of the single beer, you can make a new call to the API: 

`https://ih-beers-api2.herokuapp.com/beers/:id`